emergence_game/src/main.rs

```diff
@@ -6,7 +6,7 @@ fn main() {
         .add_plugins(DefaultPlugins.set(WindowPlugin {
             window: WindowDescriptor {
                 title: "Emergence".to_string(),
-                present_mode: PresentMode::Immediate,
+                present_mode: PresentMode::AutoNoVsync,
```

Comment:
This choice could use a comment.

Reply:
Because of this section of the description of `PresentMode::Immediate`: ["Selecting this variant will panic if not supported, it is preferred to use PresentMode::AutoNoVsync."](https://docs.rs/bevy/latest/bevy/window/enum.PresentMode.html#:~:text=optimal%20for%20mobile.-,Selecting%20this%20variant%20will%20panic%20if%20not%20supported%2C%20it%20is%20preferred%20to%20use%20PresentMode%3A%3AAutoNoVsync.,-Mailbox)